data/map.txt

```diff
@@ -18250,6 +18306,8 @@ system Pantica
 	attributes "ember waste"
 	habitable 486.68
 	belt 1203
+	hazard "EW Base Heat" 100
+	hazard "EW Base Storm" 25000
```

Comment:
For reference on the last comment, those systems with a lowered spawn rate of the storm will have a storm 900/25000 = 3.6% of the time.

Reply:
That sounds good to me.